sys/include/net/fib.h

```diff
@@ -11,6 +11,8 @@
  * @ingroup     net
  * @brief       FIB implementation
  *
+ *              This module is unused by RIOT's networking stacks,
+ *              see @ref net_gnrc_ipv6_nib_ft instead.
```

Comment:
Why the indent? Since this is going into the `@detail` part of the doc anyways, I'd prefer (but not block on)

```suggestion
 * This module is unused by RIOT's networking stacks, see @ref net_gnrc_ipv6_nib_ft instead.
```